docs/orchestrating-elastic-stack-applications/beat.asciidoc

```diff
@@ -203,9 +203,12 @@ You can specify the Beat to deploy and its version through `type` and `version`
 
 ECK supports the deployment of any Community Beat. `type` and `version` specification elements have to be provided. In addition:
 
-- `image` element in the specification must point to the image to be deployed
--  a role must exist in Elasticsearch and have the permissions required by the Beat. The role name must be `eck_beat_$type_role`, where `$type` is the Beat type. For example, when deploying `type: kafkabeat`, the role name would be `eck_beat_kafkabeat_role`. See the link:https://www.elastic.co/guide/en/elasticsearch/reference/current/defining-roles.html[Elasticsearch documentation] for more details.
+1. `image` element in the specification must point to the image to be deployed
+2. the following roles must exist in Elasticsearch:ave the permissions required by the Bea
```

Comment:
```suggestion
2. the following roles must exist in Elasticsearch and have the permissions required by the Beat:
```

Reply:
Thanks! For some reason it didn't get to the first fix commit, fixed now.